scripts/exregional_nonvarcldanl.sh

```diff
@@ -172,13 +172,13 @@ cp_vrfy ${fixgriddir}/fv3_akbk                               fv3_akbk
 cp_vrfy ${fixgriddir}/fv3_grid_spec                          fv3_grid_spec
 
 if [ -r ${bkpath}/phy_data.nc ]; then  # Use background from restart
-  cp_vrfy ${bkpath}/fv_core.res.tile1.nc         fv3_dynvars
-  cp_vrfy ${bkpath}/fv_tracer.res.tile1.nc       fv3_tracer
+  ln_vrfy  -snf  ${bkpath}/fv_core.res.tile1.nc         fv3_dynvars
+  ln_vrfy  -snf  ${bkpath}/fv_tracer.res.tile1.nc       fv3_tracer
   ln_vrfy -s ${bkpath}/sfc_data.nc                  fv3_sfcdata
 else                                   # Use background from input (cold start)
   ln_vrfy -s ${bkpath}/sfc_data.tile7.halo0.nc      fv3_sfcdata
-  cp_vrfy ${bkpath}/gfs_data.tile7.halo0.nc      fv3_dynvars
-  cp_vrfy ${bkpath}/gfs_data.tile7.halo0.nc      fv3_tracer
+  ln_vrfy  -snf  ${bkpath}/gfs_data.tile7.halo0.nc      fv3_dynvars
+  ln_vrfy  -snf  ${bkpath}/gfs_data.tile7.halo0.nc      fv3_tracer
```

Comment:
The above 175-181 changes will apply to all applications, not only for ensemble runs. I think cloud analysis is not ready to run for the RRFS system as it degraded the short term forecasts. @hu5970 Am I correct?

I have a change to switch between ln_vrfy and cp_vrfy depending the "cloud analysis" option in the config file, but it is not contributed back yet.  Maybe I can commit it back first.

Reply:
The cloud analysis is not ready for real-time test. I made lots of updates on both code and scripts recently.
But I think it is OK to make a link because the default assumption is we want cloud analysis to make impact on the forecast if we turn it on in configuration. We will manually turn it off for the further testing. I also suggest "do not add more option on cloud analysis". We should have usable cloud analysis soon.

Reply:
This is not turned on unless you have that option in config.sh. And currently it is turned off by default so it won't affect the other runs, I believe. 
If you can have that option to switch between ln_vrfy and cp_vrfy in config.sh, that would be great!
In the original cloudanalysis script, it only copy over the files to the workdir but it doesn't copy the updated files back to INPUT/. This "ln_vrfy" option was to follow other DA set up.

Reply:
> The cloud analysis is not ready for real-time test. I made lots of updates on both code and scripts recently. But I think it is OK to make a link because the default assumption is we want cloud analysis to make impact on the forecast if we turn it on in configuration. We will manually turn it off for the further testing. I also suggest "do not add more option on cloud analysis". We should have usable cloud analysis soon.

Looks fine to me if it will not bother real-time RRFS runs.  So is the problem solved that a very short integration after cloud analysis will generate too much cloud conversion? 

My original thought is add a kind of "yes_and_no" choice to the "do_nonvar_cldanal" option for those who wants to run cloud analysis but don't want to update the forecast IC. It will NOT add an extra option but utilize the existed "do_nonvar_cldanal" option. It works in my tests.
@hu5970